cmd/triggerrun/cmd/root.go

```diff
@@ -88,24 +93,57 @@ func trigger(triggerFile, httpPath string) error {
 		return fmt.Errorf("fail to get clients: %w", err)
 	}
 
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return fmt.Errorf("Failed to build config from the flags: %w", err)
+	}
+
 	logger, _ := zap.NewProduction()
 	sugerLogger := logger.Sugar()
 	eventID := template.UID()
+	sinkClients, err := sink.ConfigureClients(config)
```

Comment:
A lot of the logic here is the same as https://github.com/tektoncd/triggers/blob/master/cmd/eventlistenersink/main.go#L52

It would be nice to have a NewSink function in the sink package that returns a Sink struct. The NewSink function should take in the config *rest.Config as an input and create a Sink with the right clients/logger.